code/game/objects/effects/contraband.dm

```diff
@@ -32,6 +33,14 @@
 		//If the poster structure is being deleted something has gone wrong, kill yourself off too
 		RegisterSignal(poster_structure, COMSIG_PARENT_QDELETING, .proc/react_to_deletion)
 
+/// Adds contextual screentips
+/obj/item/poster/add_context(atom/source, list/context, obj/item/held_item, mob/user)
+	. = ..()
+	if (!istype(held_item, /obj/item/shard))
+		return .
+	context[SCREENTIP_CONTEXT_LMB] = "Booby trap poster"
+	return CONTEXTUAL_SCREENTIP_SET
```

Comment:
This should use `/datum/element/contextual_screentip_item_typechecks` instead.

https://github.com/tgstation/tgstation/blob/master/code/datums/elements/screentips/README.md#datumelementcontextual_screentip_item_typechecks